platform/commonUI/formats/src/UTCTimeFormat.js

```diff
@@ -34,6 +34,18 @@ define([
             "YYYY-MM-DD"
         ];
 
+    var MS = 1,
```

Comment:
Ah, good catch. They were used until very recently when I changed the implementation :)